rmf_traffic/test/unit/test_BroadPhase.cpp

```diff
@@ -18,6 +18,8 @@
 #include <rmf_utils/catch.hpp>
 
 #include <rmf_traffic/Conflict.hpp>
+#include <rmf_traffic/geometry/SimplePolygon.hpp>
```

Comment:
It looks like the `SimplePolygon` header was added to the test file but never actually tested.

Reply:
Thanks for catching that. I've put the tests for characteristic lengths in `make_shapes.cpp`